torntools/scripts/content/racing/ttRacingUpgrades.js

```diff
@@ -160,3 +167,11 @@ function resetUpgrades() {
 		}
 	}
 }
+
+function winPercentage() {
+	if (doc.find(".tt-win-percentage")) return;
+	doc.findAll(".enlist-info").forEach((stat)=> {
+		let values = stat.find(".enlisted-stat").innerText.split("• ").filter((x) => x.trim().includes("Races")).map((x) => x.split(":")[1].trim())
+		stat.find(".enlisted-stat").insertAdjacentHTML("beforeEnd", `<li class="tt-win-percentage">• Win Percentage: ${(values[0]/values[1]).toFixed(2)*100}% </li>`);
```

Comment:
`(..).toFixed(2)` is great but after multiplying by 100, toFixed should be taken again just in case.

![image](https://user-images.githubusercontent.com/20304303/115507884-5aa45000-a285-11eb-9702-3e5c61cca597.png)

Reply:
Also should count for "Races entered: 0". (division by zero)

![image](https://user-images.githubusercontent.com/20304303/115508085-99d2a100-a285-11eb-869d-444def461f11.png)